website/docs/contributing/writing-guide.md

```diff
@@ -92,18 +92,18 @@ For MongoDB shell commands, use `js` language.
 Our tooling will automatically reformat those blocks.
 
 ```js
-db.league.find({ club: 'PSG' })
+db.league.find({ club: "PSG" })
 ```
 
 For MongoDB shell results, use `json5` language and copy&paste the output as-is,
-with unquoted field names, single quotes for strings, without trailing commas, etc.
-Our tooling will not reformat those blocks.
+with unquoted field names, without trailing commas, etc.
+Our tooling will automatically reformat those blocks.
```

Comment:
Are `js` or `json5` formatted the same way or are there differences in formatting? What's the reason for separating these?